apps/cowswap-frontend/src/modules/twap/pure/PrimaryActionButton/getTwapFormState.tsx

```diff
@@ -21,18 +21,18 @@ export interface TwapFormStateParams {
 
 export enum TwapFormState {
   LOADING_SAFE_INFO = 'LOADING_SAFE_INFO',
-  NOT_SAFE = 'NOT_SAFE',
+  NOT_TX_BUNDLE_SUPPORT = 'NOT_TX_BUNDLE_SUPPORT',
```

Comment:
Nitpick. Doesn't sound right.

Maybe `NO_TX_BUNDLE_SUPPORT` or `TX_BUNDLING_NOT_SUPPORTED`

Reply:
Thanks!
Renamed to TX_BUNDLING_NOT_SUPPORTED